LiveSplit.AutoSplitters.xml

```diff
@@ -15566,32 +15566,34 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Red Ball</Game>
+            <Game>Red Ball 5</Game>
+            <Game>Red and Blue Balls</Game>
+            <Game>Red and Blue Balls 2</Game>
+            <Game>Red and Blue Balls 3</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rb1improved.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
         </URLs>
         <Type>Script</Type>
         <Description>Perfectly splits upon completing each level. Click "Website" to download the game. (By Maximum)</Description>
         <Website>https://drive.google.com/file/d/12X0J05rZ6gRnPgybDJuz56xGGnxjaOev/view?usp=sharing</Website>
```

Comment:
Is this still the correct download link, now that you've changed which game the autosplitter is for?

Reply:
Yes, the side games still use the old autosplitter (Red Ball 5 + the Red and Blue Balls games)

Reply:
I was asking about the Google Drive link